noir-projects/noir-protocol-circuits/crates/types/src/constants.nr

```diff
@@ -173,7 +173,7 @@ pub global DEPLOYER_CONTRACT_INSTANCE_DEPLOYED_MAGIC_VALUE =
 // GAS DEFAULTS
 pub global DEFAULT_GAS_LIMIT: u32 = 1_000_000_000;
 pub global DEFAULT_TEARDOWN_GAS_LIMIT: u32 = 12_000_000;
-pub global MAX_L2_GAS_PER_ENQUEUED_CALL: u32 = 12_000_000;
+pub global MAX_L2_GAS_PER_TX_PUBLIC_PORTION: u32 = 12_000_000;
```

Comment:
Is the _PUBLIC_PORTION necessary? gas only makes sense in public, right?

Reply:
Not quite. There is gas used up by private execution before things get to public